template/README.md

```diff
@@ -463,6 +464,62 @@ if (process.env.NODE_ENV !== 'production') {
 }
 ```
 
+## Fetching AJAX Requests
+
+This project includes a [fetch polyfill](https://github.com/github/fetch), which makes it easier to make web requests.
+
+The `fetch` function allows to easily makes AJAX requests. It takes in a URL as an input and returns a `Promise` that resolves to a `Response` object. You can find more information about `fetch` [here](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch).
```

Comment:
Let's say "the global fetch function" instead to make it clear you don't need to import it.